src/Site/views/languageforge/container/languageforge.html.twig

```diff
@@ -151,28 +151,6 @@
     {% endif %}
 {% endblock %}
 
-{% block footer %}
-    <footer class="footer">
-        <div class="d-flex">
-            <div class="d-none d-sm-flex align-items-end">
-                <div>
-                    <a href="https://inter.payap.ac.th/"><img src="/Site/views/shared/image/payap_logo.png"
-                                                          alt="Payap University Logo" width="85" height="25" /></a>
-                    <a href="http://www.sil.org"><img src="/Site/views/shared/image/sil_logo_small.png"
-                                                      alt="SIL International Logo" width="25" height="30" /></a>
-                </div>
-            </div>
-            <div class="d-flex flex-grow justify-content-end align-items-end">
-                <small>
-                    v {{ version }}.
-                    Copyright <span class="notranslate">{{ "now"|date("Y") }}</span> <a href="http://www.sil.org" class="links">SIL International</a>.
-                    <a href="/terms_and_conditions" class="links">Terms and conditions</a>.
```

Comment:
it occurred to me after approval that anything left in the repo that didn't move to another place in the app should probably also be removed...